demo/demo.js

```diff
@@ -4,6 +4,8 @@ document.addEventListener("DOMContentLoaded", () => {
   const videoElement = document.querySelector("video");
   const videoEventFilter = new VideoEventFilter(videoElement);
   videoEventFilter.addEventListener("*", (event, data) => {
-    console.log("EVENT:", event);
+    console.log("event", event, "data", data);
   });
+
+  window.videoEventFilter = videoEventFilter;
```

Comment:
Any reason to put it on the window object? 🤔

Reply:
Basically to allow for testing certain aspects of the code during development, nice to have 🤷‍♂️